### ISSUE TITLE
UI Fieldtypes

### ISSUE THREAD

**Issue by jasonvarga** [statamic/ideas#357]
We want to have fieldtypes that get rendered in the editor but don't save any data.

For instance, the "section" fieldtype. It's just a header, but we use some relatively hacky css and templating to make sure it renders differently. It also has its data (or lack of data) tracked in the Vuex store.

In addition to the section fieldtype, there could be html (which also already exists), information blocks, etc.

These UI only fieldtypes would have different logic for rendering which would exclude form labels, etc.

This could also be useful for the forms feature, since we allow you to loop over your blueprint fields. See statamic/cms#2575

(I was sure that an issue already existed for this, but I can't find it so this might be a duplicate)

**Comment by robdekort** [comment]
I love this idea. The ability to group fields in the future would be a welcome feature as well. I remember another issue as well where fieldgroups were discussed.

**Comment by jelleroorda** [comment]
I’d also like to coin a specific addition of what Jason mentions here. Right now I regularly need to add data to an entry that should not be seen by the editors (because it gets set programmatically). Right now I create a text field, set the `input_type` to `hidden` and a `display` of ‘ ‘. This does however take up “empty space” where you position the field in the blueprint. It would be nice to have a field that doesn’t “update” it’s data through the CP like mentioned, but in addition doesn’t render anything at all.

**Comment by sjclark** [comment]
This would also be useful for the **HTML fieldtype** which currently displays "HTML" as the title by default. Found a way around by setting `display: ' '` directly in my Blueprint YAML.

**Comment by jasonvarga** [comment]
Yup, the html fieldtype is mentioned in the original description. 👍

**Comment by helloDanuk** [comment]
A «Force a new line»-option https://github.com/statamic/ideas/issues/96 might be related to this idea. Kind of a UI helper. We could improve readability once a few narrow fields are used.

**Comment by jackmcdade** [comment]
We have pretty much everything mentioned here now. ❤️
